### PR TITLE
Jb49383

### PR DESCRIPTION
**Pull request by martyone**

### CODE REVIEW

**Comment by martyone** [comment]
Just the very last commit belongs to this PR

File: src/libs/sfdk/dockervirtualmachine.cpp [hunk 1]
```diff
@@ -562,11 +564,19 @@ QStringList DockerVirtualMachinePrivate::makeCreateArguments() const
 
     QStringList arguments;
     arguments.append("create");
-    arguments.append("--privileged");
-    arguments.append("--cap-drop=NET_ADMIN");
+    arguments.append("--env=container=docker");
```
Comment (vigejolla) [inline src/libs/sfdk/dockervirtualmachine.cpp]: What is this needed for?

Comment (martyone) [reply src/libs/sfdk/dockervirtualmachine.cpp]: for systemd

**Comment by vigejolla** [comment]
LGTM

**Pull request merged**